apps/researcher/src/messages/nl/messages.json

```diff
@@ -299,20 +293,23 @@
     "title": "Voeg een verhaal toe",
     "description": "Verhaal",
     "descriptionSubTitle": "Hier kunt u uw eigen verhaal toevoegen aan de beschrijving van dit object.",
-    "community": "Community",
-    "communitySubTitle": "Selecteer de community waarvoor u spreekt.",
     "citation": "Bron",
-    "citationSubTitle": "Geef informatie op die uw verhaal ondersteunt.",
+    "citationSubTitle": "Geef informatie die uw verhaal ondersteunt.",
     "inLanguage": "Taal",
-    "languageSubTitle": "Selecteer een taal voor het verhaal. Begin met typen om een taal te selecteren.",
+    "languageSubTitle": "Selecteer de taal van het verhaal. Begin met typen om een taal te selecteren.",
     "successfullyAdded": "Uw verhaal is toegevoegd aan het object. Het kan even duren voordat het zichtbaar is.",
-    "serverError": "Er is iets misgegaan tijdens het maken van het verhaal. Probeer het later opnieuw.",
+    "labelAttributionId": "Persoonlijke profiel-URL",
+    "descriptionAttributionId": "Door uw verhaal toe te voegen, wordt uw persoonlijke URL opgeslagen bij het verhaal. Dit kan een link zijn naar uw persoonlijke LinkedIn, Facebook of ORCID-profiel. Het is mogelijk niet zichtbaar voor andere bezoekers, maar de gegevens worden opgeslagen in de cloud.",
```

Comment:
Minor:
1. `LinkedIn-, Facebook- of ORCID-profiel`
2. `...maar de gegevens...` --> is this the 'persoonlijke URL'? If so, use a singular form?